utils/src/main/kotlin/de/cyface/app/utils/trips/DetailsFragment.kt

```diff
@@ -137,10 +137,11 @@ class DetailsFragment : Fragment() {
         // Chart
         val chart = root.findViewById(R.id.chart) as LineChart
         val altitudes = persistence.loadAltitudes(measurementId)
-        if (altitudes == null || altitudes.isEmpty()) {
+        if (altitudes.isNullOrEmpty()) {
             binding.elevationProfileTitle.text = getString(R.string.elevation_profile_no_data)
             chart.visibility = GONE
         } else {
+            // We could also show the relative elevation profile (starting at elevation 0)
```

Comment:
that was a suggestion from you in an email conversation, so I added this as comment here in case we have issues with the current implementation and / or change this in the future.